src/components/Desktop/ContextMenu.svelte

```diff
@@ -37,7 +37,7 @@
   <div
     class="container"
     class:dark={$theme.scheme === 'dark'}
-    style:transform="translate({xPos}px, {yPos}px);"
```

Comment:
HI, can you revert to using the style directive, and remove the `;` from the end of the line? That's the real culprit

Reply:
Yep, reverted and removed `;`